potatobot/potatobot.py

```diff
@@ -102,15 +104,19 @@ def _handle_single_post(self, post):
         post: The piazza_api post object.
 
         """
-        post_info = post["history"][-1]
-        post_username_id = post_info["uid"]
-        post_text = post_info["content"]
-
+        post_history = post["history"][-1]
+        post_username_id = post_history["uid"]
+        post_text = post_history["content"]
+        post_status = post["status"]
```

Comment:
Split this block into paragraphs. It might be that we don't need `post_username_id` in its own variable, for one thing. We can talk in person.